docs-translations/zh-CN/README.md

```diff
@@ -0,0 +1,58 @@
+# Jupyter Notebook
+
+[![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/jupyter)
+[![Build Status](https://travis-ci.org/jupyter/notebook.svg?branch=master)](https://travis-ci.org/jupyter/notebook)
+[![Documentation Status](https://readthedocs.org/projects/jupyter-notebook/badge/?version=latest)](https://jupyter-notebook.readthedocs.io/en/latest/?badge=latest)
+                
+英文版链接 : [[English Version](http://github.com/jupyter/notebook/)]
+
+Jupyter Notebook是一个基于Web的交互式交互式笔记本环境。
+
+![Jupyter notebook example](resources/running_code_med.png "Jupyter notebook example")
+
```

Comment:
same as previous regarding the Notice block.

Reply:
I'm sorry, i'm not entirely sure what I have to do.
I'm new to this sort of thing and I couldn't find what Notice block meant.
Again, I apologize but could you briefly explain what exactly I have to do

Reply:
No worries.  
These translations are translations of the README.md file.  We've recently added a section in the README.md labelled "Notice", just following the Notebook image and preceding the heading "Jupyter notebook, the language-agnostic evolution of IPython notebook".  Your translations don't appear to contain that block of text and I think it would be a good idea to include that information since it's important to users and contributors.  I included a link to this section of the README in the previous comment above, and [here as well](https://github.com/jupyter/notebook#notice).
Thank you.

Reply:
Sorry for all this time, I had my exams in school

Reply:
No worries - I hope your exams went well.  Will you be updating the Chinese translation as well?  No rush.

Reply:
I have updated both the documents